tests/e2e/workloads/couchbase/test_couchbase.py

```diff
@@ -2,33 +2,18 @@
 import pytest
 
 from ocs_ci.framework.testlib import E2ETest, workloads
-from ocs_ci.ocs.couchbase import CouchBase
 
 log = logging.getLogger(__name__)
 
 
-@pytest.fixture(scope='function')
-def couchbase(request):
-
-    couchbase = CouchBase()
-
-    def teardown():
-        couchbase.teardown()
-    request.addfinalizer(teardown)
-    return couchbase
-
-
 @workloads
 @pytest.mark.polarion_id("OCS-785")
 class TestCouchBaseWorkload(E2ETest):
     """
     Deploy an CouchBase workload using operator
     """
-    def test_cb_workload_simple(self, couchbase):
+    def test_cb_workload_simple(self, couchbase_factory_fixture):
         """
         Testing basic couchbase workload
         """
-        couchbase.setup_cb()
-        couchbase.create_couchbase_worker(replicas=5)
-        couchbase.run_workload(replicas=5)
-        couchbase.analyze_run()
+        couchbase_factory_fixture(replicas=3)
```

Comment:
Please export data to the google sheet in this basic CB test.

Reply:
couchbase_factory_fixture() is handling export data as well

Reply:
Yes, this was my continuation comment to above :)
Remove the gsheet export from the fixture and handle it in this test case.